lib/uplink/project.go

```diff
@@ -200,8 +200,3 @@ func (p *Project) OpenBucket(ctx context.Context, bucketName string, access *Enc
 		streams:      streamStore,
 	}, nil
 }
-
-// Close closes the Project.
-func (p *Project) Close() error {
```

Comment:
why its moved to `uplink.go`?

Reply:
Because the constructing of project is there, so it's easier to see what needs to be closed. However, the right approach probably is to move the open project into `project.go`... but since that's a larger move, I opted for that cleanup later.

Reply:
make sense, maybe TODO about it?

Reply:
done